src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -44,17 +40,6 @@ const getRecommendations = (service, assetUri) => {
   }
 
   return [
-    {
-      name: 'recommendations',
-      attachAgent: true,
-      path: getRecommendationsUrl({
-        assetUri,
-        engine: 'unirecs_camino',
-      }),
-      assetUri,
-      api: 'recommendations',
-      apiContext: 'secondary_data',
-    },
```

Comment:
I think this will need to stay and just the `engine:` changed to `unirecs_datalab` like above, otherwise the Portuguese/Brasil service won't have the base `recommendations` object which is used when there is no experiment variant.

Reply:
Ah ok. I mistakenly thought that `datalabContentRecommendations` was going to be the default for the Brasil experiment

Reply:
No probs! Thanks for fixing :)